src/Jackett.Common/Definitions/hddolby.yml

```diff
@@ -166,5 +189,8 @@ search:
       text: 1.2
     description:
       selector: td.rowfollow:nth-child(2)
-      remove: a, img
+      remove: a, b, font, img, span
+      filters:
+        - name: replace
+          args: [" 剩余时间：", ""]
 # NexusPHP Standard v1.5 Beta 4
```

Comment:
you may want to check that if the site allows profile language to be set to english then this may be 'will end in'